### DISCUSSION TITLE
윤현영 1회차 블로그 피드백 요청드립니다🙏

### DISCUSSION THREAD

**Discussion by hyyun1113** [Orm-camp/blog-feedback#10] [동글동글 2기 피드백]
[[https://learnwebdev.medium.com/오름캠프-1차-미니-프로젝트-회고-6fb05d309d5d](https://learnwebdev.medium.com/%EC%98%A4%EB%A6%84%EC%BA%A0%ED%94%84-1%EC%B0%A8-%EB%AF%B8%EB%8B%88-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%ED%9A%8C%EA%B3%A0-6fb05d309d5d)](url)

이번 미니 프로젝트 회고글 입니다. 
잘 부탁드립니다.

**Comment by juuyeoon** [comment]
현영님, 임주연 입니다!
올려주신 링크 누르면 404가 뜨는데요...!
게시물 작성하신 주소 다시 한 번 체크해 주세요 😊

**Reply by hyyun1113** [reply]
왜 404가 뜨는지 전혀 이해가 되지 않는 상황이지만.. 우선 url수정해서 재업로드 했습니다~

**Comment by taeyoung0001** [comment]
안녕하세요 현영님
역시 글 구성이랑 내용 정말 좋습니다
중간에 gif 로 넣은 센스도 훌륭하시네요! 

비록 다른 팀이였지만, 
현영님께서 쓰신 글만으로도 많이 노력하시고 열심히 하신게 보입니다.

늘 화이팅입니다!

**Comment by peppone-choi** [comment]
GIF로 이렇게 만들어서 올리는 방법도 있겠군요. 글을 쓸 때 머리가 아팠는데 앞으로는 멀티미디어도 활용 해봐야겠습니다. 많은걸 배워갑니다!

**Comment by dayoungh** [comment]
프로젝트때도 느꼈지만 현영님은 이해력이 참 뛰어나신 것 같습니다! 분명 같은 프로젝트를 진행했는 데도 어떻게 핵심만 정리해서 잘 설명하실 수 있는지,,! 제 주절주절 일기 쓰듯이 쓴 글이 부끄럽기도하고 이미지 첨부해서 보여주신 것도 좋습니다ㅎㅎㅎㅎ 더 발전한 다음 프로젝트와 회고글이 기대가 되는 글이었습니다👏👏👏
(그리고 아쉬운 점은 전혀 공감할 수 없습니다,,,😂 민폐라고 하기에는 너무 잘하셨어요!!!)

**Comment by ggaong** [comment]
회고글 작성하시느라 고생 많으셨습니다😀
gif랑 이미지를 넣어서 구현 기능을 보여주는 부분이 눈에 잘 들어오는 느낌이라 다음 회고 글에 참고해서 작성 해보고 싶네요!
다음 프로젝트도 화이팅입니당 👍👍

**Comment by euneunsol** [comment]
현영님! 
회고글 중간에 실행 화면을 영상으로 추가하신 기술이 궁금합니다!
실행 화면을 여러 사진으로 나열하는 것 보다 영상의 형태로 보여주는 것이 무엇을 개발했는지 한눈에 들어와서 좋았습니다!
좋은 글 감사합니다!
